app/src/main/res/values/strings.xml

```diff
@@ -130,4 +130,10 @@
     <string name="drawer_turn_shift">Entrega de turno</string>
     <string name="drawer_pre_operational">Preoperacional</string>
     <string name="drawer_hceud">HCEUD</string>
+    <string name="drawer_device_auth">Asociar dispositivo al vehículo</string>
+    <string name="drawer_signature_and_fingerprint">Registrar firma y huella</string>
+
+    <string name="drawer_header_zone">Subred: %s</string>
+    <string name="drawer_header_provider">Base: %s</string>
+    <string name="drawer_header_functional_unit">Unidad funcional</string>
```

Comment:
Can we move this resources on top of the role ones? Move to the top of the <!-- Drawer Menu --> section of strings


```suggestion
    <string name="drawer_header_zone">Subred: %s</string>
    <string name="drawer_header_provider">Base: %s</string>
    <string name="drawer_header_functional_unit">Unidad funcional</string>
    
    <string name="drawer_role_doctor">Médico</string>
    <string name="drawer_role_aux">Auxiliar</string>
    <string name="drawer_role_driver">Conductor</string>
```